Detectors/MUON/MCH/Workflow/CMakeLists.txt

```diff
@@ -122,30 +96,6 @@ o2_add_executable(
         COMPONENT_NAME mch
         PUBLIC_LINK_LIBRARIES O2::Framework O2::DetectorsBase O2::DataFormatsMCH O2::MCHTracking)
 
-o2_add_executable(
-        tracks-sink-workflow
-        SOURCES src/TrackSinkSpec.cxx src/tracks-sink-workflow.cxx
-        COMPONENT_NAME mch
-        PUBLIC_LINK_LIBRARIES O2::Framework O2::DataFormatsMCH O2::MCHBase)
-
-o2_add_executable(
-        tracks-writer-workflow
-        SOURCES src/tracks-writer-workflow.cxx
-        COMPONENT_NAME mch
-        PUBLIC_LINK_LIBRARIES O2::MCHWorkflow)
-
-o2_add_executable(
-        tracks-reader-workflow
-        SOURCES src/tracks-reader-workflow.cxx
-        COMPONENT_NAME mch
-        PUBLIC_LINK_LIBRARIES O2::MCHWorkflow)
-
-o2_add_executable(
-        tracks-sampler-workflow
-        SOURCES src/TrackSamplerSpec.cxx src/tracks-sampler-workflow.cxx
-        COMPONENT_NAME mch
-        PUBLIC_LINK_LIBRARIES O2::Framework O2::DataFormatsMCH O2::MCHBase)
-
```

Comment:
you are a step ahead in the removal of the track IO workflows ;-)

Reply:
indeed ;-) fixed.

Reply:
not completely. You only put back 2 out of 5 track related workflows.

Reply:
ok, let me try again...